storage/innobase/xtrabackup/src/quicklz/quicklz.h

```diff
@@ -62,7 +62,7 @@ typedef unsigned short int ui16;
 
 // Detect if pointer size is 64-bit. It's not fatal if some 64-bit target is not
 // detected because this is only for adding an optional 64-bit optimization.
-#if defined _LP64 || defined __LP64__ || defined __64BIT__ || _ADDR64 || \
+#if defined _LP64 || defined __LP64__ || defined __64BIT__ || defined _ADDR64 || \
```

Comment:
I see same file exists in 2.4? is it impacting build failure for 2.4 as well?

Reply:
Interestingly, the compile succeeds on 2.4 (no warnings)

Reply:
may be some dependency version. I think better to add fix in 2.4 as well

Reply:
Fixed in 2.4 also.  https://github.com/percona/percona-xtrabackup/pull/1113